docs/docsite/rst/network_getting_started.rst

```diff
@@ -0,0 +1,246 @@
+
+Prerequisites
+-------------
+
+* Ansible 2.3 (or higher) installed :doc:`intro_installation`
+* One or more network device compatible with Ansible
+* Basic understanding of YAML :doc:`YAMLSyntax`
+* Basic understanding of Jinja2 Templates :doc:`playbooks_templating`
+
+.. FIXME FUTURE Gundalow - Once created we will link to the connection table here (which platforms support network_cli & credentials through inventory)
+.. FIXME FUTURE Gundalow -  Using ``ansible_ssh_pass`` will not work for REST transports such as ``eapi``, ``nxapi`` - Once documented in above FIXME add details her
+
+Audience
+--------
+
+* Basic Linux (comfortable on the command line)
+* Basic Network switch & router configuration knowledge
+
+
+Solution
+=========
+
+Create an inventory file.
+
+
+**Inventory file**
+
+Create a file called ``inventory``, containing:
+
+.. code-block::
+
+   [switches:children]
+   ios
+   vyos
+
+   [ios]
+   ios01.example.net ansible_connection=local ansible_user=cisco ansible_ssh_pass=cisco
+
+   [vyos]
+   vyos01.example.net ansible_connection=local ansible_user=admin ansible_ssh_pass=admin
+
+
+**Playbook**
+
+Create a file called ``fetch-facts.yml`` containing the following:
+
+.. code-block:: yaml
+
+   - name: "Download switch configuration"
+     hosts: switches
+     gather_facts: no
+
+     tasks:
+       ###
+       # Collect data
+        - name: Gather facts (ios)
+          ios_facts:
+          when: "'ios' in group_names"
+
+        - name: Gather facts (vyos)
+          vyos_facts:
+          when: "'vyos' in group_names"
+
+        ###
+        # Demo variables
+        #
+        - name: Display some facts
+          debug:
+            msg: "The hostname is {{ ansible_net_hostname }} and the OS is {{ ansible_net_version }}"
+
+        - name: Display facts from a specific host
+          debug:
+            var: hostvars['vyos01.example.net']
+
+        - name: Write facts to disk using a template
+          copy:
+            content: |
+              IOS device info:
+                {% for host in groups['ios'] %}
+                Hostname: {{ hostvars[host].ansible_net_version }}
+                Version: {{ hostvars[host].ansible_net_version }}
+                Model: {{ hostvars[host].ansible_net_model }}
+                Serial: {{ hostvars[host].ansible_net_serialnum }}
+                {% endfor %}
+
+              VyOS device info:
+                {% for host in groups['vyos'] %}
+                Hostname: {{ hostvars[host].ansible_net_version }}
+                Version: {{ hostvars[host].ansible_net_version }}
+                Model: {{ hostvars[host].ansible_net_model }}
+                Serial: {{ hostvars[host].ansible_net_serialnum }}
+                {% endfor %}
+            dest: /tmp/switch-facts
+          run_once: yes
+
+        ###
+        # Get running configuration
+        #
+
+        - name: Backup switch (ios)
```

Comment:
Can we have config playbook, separately. As discussed during the contributor summit?

config -> set the hostname
command -> do a show version.
facts -> above playbook is good.

Reply:
I was going to do those three in a separate document.

Reply:
Thanks, thats fine.